README.md

```diff
@@ -0,0 +1,54 @@
+## Temporarily lock files and folders on your reMarkable
+I often "forget" the time when reading a great book. In the past there was nothing that could be done about that. In the time of e-readers however, we can. _Book safe_ hides the content of one or more folders from the reMarkable UI within a given time period, and adds a pdf that lists what has been blocked. While folders are blocked, the cloud sync is disabled by default, to prevent deleting and re-uploading of files.
```

Comment:
at the end I would change this to: to prevent the cloud sync from treating hidden files as deleted, re-uploading them when they re-appear.